library/mlops/exceptions/tracking.py

```diff
@@ -0,0 +1,6 @@
+
+def project_id_missing_exception():
```

Comment:
To bo honest, project_id_missing_exception() is ambiguous. Firstly, project_id is None and secondly project with this id does not exist. In my opinion this should be split into two exceptions: project_id_not_specified and project_id_not_found.